blog/bff-case-study-2024-08-30.md

```diff
@@ -37,6 +33,20 @@ As a former employees, We're thrilled to share how we harnessed the power of Gra
 
 ## Case Study: Dream11's GraphQL Journey
 
+Before we dive into the details of our GraphQL journey, check out this video for a high-level overview of how we tackled the challenges and transformed our backend architecture:
```

Comment:
Doesn't gel well here. I think it should be where call to action was,
also disable suggestions as we have done here
https://tailcall.run/docs/

Reply:
Thanks for the suggestion!
Unfortunately, it's not possible to completely remove video suggestions in YouTube embeds. Even after `rel=0` param, YouTube will still show related videos, but only from the same channel as the embedded video.

Reply:
In case of https://tailcall.run/docs/ , the reason we are not seeing any suggestions is because https://www.youtube.com/@tusharmath has only one video.